### PR TITLE
docs(readme): update lighthouse-mocha-example repo and description

### PR DESCRIPTION
**Pull request by rishichawda**
**Summary**

Older example with mocha doesn't work with current lighthouse version (older example is based on v1). This is the updated example repository link with a working example of lighthouse tests with mocha and chai.

**Is this a bugfix, feature, refactoring, build related change, etc?**

This is a documentation related change.

**Describe the need for this change**

Older example doesn't work since it is based on an older version of lighthouse and is no longer maintained.

**Related Issues/PRs**

None.

### CODE REVIEW

**Comment by rishichawda** [comment]
I signed it!

File: readme.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ Other awesome open source projects that use Lighthouse.
 * **[lighthouse-hue](https://github.com/ebidel/lighthouse-hue)** - set the color of Philips Hue lights based on a Lighthouse score
 * **[lighthouse-lambda](https://github.com/joytocode/lighthouse-lambda)** - Run Lighthouse on AWS Lambda with prebuilt stable desktop Headless Chrome.
 * **[lighthouse-magic-light](https://github.com/manekinekko/lighthouse-magic-light)** set the color of the MagicLight Bluetooth Smart Light Bulb based on Lighthouse score
-* **[lighthouse-mocha-example](https://github.com/justinribeiro/lighthouse-mocha-example)** - gather performance metrics via Lighthouse and tests them in Mocha
+* **[lighthouse-mocha-example](https://github.com/rishichawda/lighthouse-mocha-example)** - Run lighthouse performance tests with Mocha and chrome-launcher
```
Comment (Hoten) [inline readme.md]:
```suggestion
* **[lighthouse-mocha-example](https://github.com/rishichawda/lighthouse-mocha-example)** - Run Lighthouse performance tests with Mocha and chrome-launcher
```

**Comment by rishichawda** [comment]
@Hoten looks like there's some issue with CLA on your account after I accepted the commit suggestion that you added.

**Comment by justinribeiro** [comment]
Yeah, I've been meaning to update the older mocha example to be closer inline with my more robust Jest example (https://github.com/justinribeiro/lighthouse-jest-example).

I'm fine with replacing it, though the name thing might throw people (that example and my old one aren't exactly the same concept-wise, but probably splitting hairs).

**Comment by rishichawda** [comment]
@brendankenny @justinribeiro I have no issue with renaming it to something different. Just kept as it is since the name felt relevant to what is in the repository.

**Pull request merged**